### PR TITLE
[IMP] social, social_*: clean and improve UX in social app

### PR DESCRIPTION
**Pull request by dbeguin**

### CODE REVIEW

File: addons/sale/models/utm_campaign.py [hunk 1]
```diff
@@ -7,8 +7,10 @@ class UtmCampaign(models.Model):
     _inherit = 'utm.campaign'
     _description = 'UTM Campaign'
 
-    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman', compute="_compute_quotation_count")
-    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", string="Revenues generated by the campaign")
+    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman',
+        compute="_compute_quotation_count", compute_sudo=True)
+    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", compute_sudo=True,
+        string="Revenues generated by the campaign", groups='sales_team.group_sale_salesman')
```
Comment (Feyensv) [inline addons/sale/models/utm_campaign.py:13]:
```suggestion
    invoiced_amount = fields.Integer("Revenues generated by the campaign", groups='sales_team.group_sale_salesman',
        compute='_compute_sale_invoiced_amount', compute_sudo=True)
```
Do you mind applying the same order of field params for both fields and remove the useless default while you're modifying those fields? :)

**Review by Feyensv — APPROVED** [review]
Lgtm for sale scope with the current diff, what's the task-id :mag: ?

File: addons/sale/models/utm_campaign.py (hunk 1 above)
Comment (tde-banana-odoo) [reply addons/sale/models/utm_campaign.py:13]: +1 (would say: string, compute, compute_sudo, groups)

File: addons/sale/models/utm_campaign.py [hunk 2]
```diff
@@ -7,8 +7,10 @@ class UtmCampaign(models.Model):
     _inherit = 'utm.campaign'
     _description = 'UTM Campaign'
 
-    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman', compute="_compute_quotation_count")
-    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", string="Revenues generated by the campaign")
+    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman',
+        compute="_compute_quotation_count", compute_sudo=True)
+    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", compute_sudo=True,
```
Comment (tde-banana-odoo) [inline addons/sale/models/utm_campaign.py:12]: As @Feyensv says: would remove the default as compute should be complete

**Review by tde-banana-odoo — COMMENTED** [review]
Validation !

**Review by tde-banana-odoo — APPROVED** [review]

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**